langchain4j-qianfan/src/main/java/dev/langchain4j/model/qianfan/BaiduApi.java

```diff
@@ -0,0 +1,29 @@
+package dev.langchain4j.model.qianfan;
+
+import dev.langchain4j.model.qianfan.client.completion.CompletionRequest;
+import dev.langchain4j.model.qianfan.client.embedding.EmbeddingRequest;
+import dev.langchain4j.model.qianfan.client.embedding.EmbeddingResponse;
+import dev.langchain4j.model.qianfan.client.chat.ChatCompletionRequest;
+import dev.langchain4j.model.qianfan.client.chat.ChatCompletionResponse;
+import dev.langchain4j.model.qianfan.client.chat.ChatTokenResponse;
+import dev.langchain4j.model.qianfan.client.completion.CompletionResponse;
+import retrofit2.Call;
+import retrofit2.http.*;
+
+public interface BaiduApi {
+
+        @POST("rpc/2.0/ai_custom/v1/wenxinworkshop/chat/{endpoint}")
```

Comment:
Just to double check: will this URL work for all users? Somehow it feels like this is a custom URL...

Reply:
The Qianfan model does not have the modelName parameter. Different models are distinguished through endpoints, and users' private models are also distinguished through endpoints, 
all in fixed URL form:  rpc/2.0/ai_ Custom/v1/wenxinworkshop/chat/+{endpoint}
see detail : https://cloud.baidu.com/doc/WENXINWORKSHOP/s/Nlks5zkzu
![image](https://github.com/langchain4j/langchain4j/assets/35522029/1feaf285-6384-464b-826a-0fbd7a1d5524)